README.md

```diff
@@ -1,37 +1,47 @@
-# Botup
+# 𝑩𝒐𝒕𝒖𝒑 𝒃𝒚 𝟓𝟎𝟎𝒂𝒑𝒑𝒔 - 𝑪𝒉𝒂𝒕 𝑩𝒐𝒕
+
 [Get Started](https://infinity.500apps.com/botup) with with [Botup](https://botup.com)
 
-A powerful platform to build chat bots for your teams on Facebook, Slack, WhatsApp, SMS, and Websites.
-Create flows and run a bot to respond swiftly to your consumers thereby enabling powerful automation campaigns.
-As your customer base grows from various locations, there's a wide scope for business growth. Bot up has the capability to scale your operations and make sure that you respond to your customers on time.
-Botup serves best with simple setup options itself. Use the advanced features to explore the all-in-one bot which is capable of handling all your social media handle responses at one go. Enjoy the additional benefit of capturing the leads and grow your network multi fold.
+A powerful platform to build chat bots for your teams on Facebook, Slack, WhatsApp, SMS, and Websites. Create flows and run a bot to respond swiftly to your consumers thereby enabling powerful automation campaigns. As your customer base grows from various locations, there's a wide scope for business growth. Bot up has the capability to scale your operations and make sure that you respond to your customers on time. Botup serves best with simple setup options itself. Use the advanced features to explore the all-in-one bot which is capable of handling all your social media handle responses at one go. Enjoy the additional benefit of capturing the leads and grow your network multi fold.
```

Comment:
Slack integration not there yet

Reply:
Is there in apps tab but not implemented sir.

Reply:
remove it

Reply:
Removed